2996_reflection/reflection.md

```diff
@@ -6032,17 +5935,6 @@ S<V2> s2; // OK
 :::
 :::
 
-### [temp.dep.temp]{.sref} Dependent template arguments {-}
-
-Add a new paragraph to cover dependent splice template arguments.
-
-::: std
-[5]{.pnum} A template `$template-parameter$` is dependent if it names a `$template-parameter$` or if its terminal name is dependent.
-
-[[5+]{.pnum} A splice template argument is dependent if its `$splice-specifier$` is dependent.]{.addu}
```

Comment:
Don't we still need this? Cause `X<[:R:]>` is still valid if `X` has a constant template parameter?

Reply:
That's just covered by this:
https://brevzin.github.io/cpp_proposals/2996_reflection/d2996r13.html#pnum_419